### PR TITLE
[Document] update API docs and fix format

### PR DESCRIPTION
**Pull request by vankichi**
### Description:

SSIA

### Environment:

- Go Version: 1.17.7
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/user-guides/operations.md [hunk 1]
```diff
@@ -115,9 +115,14 @@ In case of using Helm and Vald's chart, please update `defaults.image.tag` field
...
-    $ kubectl replace -f https://raw.githubusercontent.com/vdaas/vald/${VERSION}/charts/vald-helm-operator/crds/valdrelease.yaml
-    $ kubectl replace -f https://raw.githubusercontent.com/vdaas/vald/${VERSION}/charts/vald-helm-operator/crds/valdhelmoperatorrelease.yaml
+```bash
+VERSION=v1.4.1
+```
+
+```bash
+kubectl replace -f https://raw.githubusercontent.com/vdaas/vald/${VERSION}/charts/vald-helm-operator/crds/valdrelease.yaml && \
+kubectl replace -f https://raw.githubusercontent.com/vdaas/vald/${VERSION}/charts/vald-helm-operator/crds/valdhelmoperatorrelease.yaml
+```
```
Comment (kmrmt) [inline docs/user-guides/operations.md:125]: Why is this split?

Comment (vankichi) [reply docs/user-guides/operations.md:125]: Because the next web page has a clipboard coy feature for code block.

**Review by kevindiu — APPROVED** [review]
LGTM

File: docs/user-guides/operations.md (hunk 1 above)
Comment (kmrmt) [reply docs/user-guides/operations.md:125]: I understand.

**Review by kmrmt — APPROVED** [review]

**Pull request merged**